qemu-check.exp

```diff
@@ -158,3 +158,6 @@ if {$::env(XEN_BOOT) == "y"} {
 	send -- "$cmd\r"
 	check_test_result $teecore
 }
+# Invoke Trusted Keys tests
+set basedir [file dirname $argv0]
+source $basedir/trusted-keys.exp
```

Comment:
Wouldn't it be more convenient to have a separate target in the makefile so that one can easily run the Trusted Keys tests on their own? (similar to `check-rust`).

Reply:
Actually the major motivation was to run Trusted Keys tests as part of CI as these are pretty quick tests which wouldn't have any noticeable impact on CI runtime. However, I liked your suggestion that one should be able to run the Trusted Keys tests standalone without invoking the `xtest`. How about we just add `check-only-trusted-keys` target that would omit the `xtest` while invoking the same script?

Reply:
Also, `check-only-xtest` can be added on similar lines?

Reply:
Sounds good to me.

Reply:
I just used additional command line option as `CHECK_TESTS` instead as that turned out to be much simpler.